Doc/reference/compound_stmts.rst

```diff
@@ -98,7 +98,7 @@ It selects exactly one of the suites by evaluating the expressions one by one
 until one is found to be true (see section :ref:`booleans` for the definition of
 true and false); then that suite is executed (and no other part of the
 :keyword:`if` statement is executed or evaluated).  If all expressions are
-false, the suite of the :keyword:`else` clause, if present, is executed.
+false, the :keyword:`else` clause's suite, if present, is executed.
```

Comment:
IMO is more redeable the original way

Reply:
@eamanu Thanks for the review. Actually the possessive form "'s" was already used inconsistently through this chapter, so this PR just uses it everywhere. And to me the possessive form is quite readable.

Reply:
Thanks for the clariftication @maggyero  :)